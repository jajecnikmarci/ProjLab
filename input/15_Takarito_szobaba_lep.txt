starttest
add room roomToClean 5 poisonous
add room room1 4
add room room2 4
add door room1 roomToClean door1
add door room2 roomToClean door2
add item roomToClean Camembert stickyCamembert
add item roomToClean Rug rug1
add person room1 Cleaner cleaner1
add person room2 Student student1
printstate roomToClean
gotoroom cleaner1 roomToClean
gotoroom cleaner1 room2
gotoroom student1 room1
gotoroom student1 roomToClean
gotoroom student1 room1
gotoroom student1 roomToClean
gotoroom student1 room1
gotoroom student1 roomToClean
pickup student1
printstate student1
gotoroom student1 room1
gotoroom student1 roomToClean
pickup student1
printstate student1
endtest
starttest 
add room room1 4
add room room2 4
add door room1 room2 door1
add person room1 Cleaner cleaner1
add person room2 Professor professor1
printstate professor1
printstate cleaner1
gotoroom cleaner1 room2
printstate professor1
printstate cleaner1
endtest
starttest 
add room room3 4
add room room4 4
add door room3 room4 door1
add person room3 Cleaner cleaner1
add person room4 Cleaner cleaner2
printstate room3
printstate cleaner1
printstate cleaner2
gotoroom cleaner1 room4
printstate cleaner1
printstate cleaner2
endtest