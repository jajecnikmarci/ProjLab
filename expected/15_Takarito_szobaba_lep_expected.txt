roomToClean
	- capacity: 5
	- people: 
	- doors: 
		door1
		door2
	- items: 
		stickyCamembert
		rug1
	- poisonEffects: 
		poisonEffects
	- stunEffects: 
	- lastCleaning: 
		StickinessEffect
student1
	- location: roomToClean
	- stunned: false
	- inventory: 
		rug1
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
student1
	- location: roomToClean
	- stunned: false
	- inventory: 
		rug1
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
professor1
	- location: room2
	- stunned: false
	- Inventory: 
	- poisonImmunities: 
cleaner1
	- location: room1
professor1
	- location: room2
	- stunned: false
	- Inventory: 
	- poisonImmunities: 
cleaner1
	- location: room2
room3
	- capacity: 4
	- people: 
		cleaner1
	- doors: 
		door1
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: 
		StickinessEffect	
cleaner1
	- location: room3
cleaner2
	- location: room4
cleaner1
	- location: room4
cleaner2
	- location: room3
